test/utils/eventLevels.test.js

```diff
@@ -261,107 +261,71 @@ describe('inRange', () => {
         expect(inRange(event, rangeStart, rangeEnd, accessors)).toBe(result)
       })
     }
-    const weekOfThe5th = [d(5), d(11)]
-    const weekOfThe12th = [d(12), d(18)]
-    ;[
+
+    const sample = [
```

Comment:
prettier was prepending a semi-colon here, and eslint was complaining. Assigning to `sample` as a workaround